webapp/src/ts/services/feedback.service.ts

```diff
@@ -76,7 +124,7 @@ export class FeedbackService {
       this.options.console[level] = (...args) => {
         const logEvent = {
           level,
-          arguments: JSON.stringify(args, getCircularReplacer()).substring(0, 1000),
+          arguments: JSON.stringify(args, getCircularReplacer()).substring(0, 5000),
```

Comment:
I had some trouble diagnosing what was happening with a truncated error of just 1000 characters. This limit was arbitrarily picked.

Reply:
Let's pull it out as a constant. 5000 is probably ok.